subsys/jwt/jwt_psa.c

```diff
@@ -24,15 +24,15 @@ int jwt_sign_impl(struct jwt_builder *builder, const unsigned char *der_key, siz
 	psa_algorithm_t alg;
 	int ret;
 
-#if defined(CONFIG_JWT_SIGN_ECDSA)
+#if defined(CONFIG_JWT_SIGN_ECDSA_PSA)
 	psa_set_key_type(&attr, PSA_KEY_TYPE_ECC_KEY_PAIR(PSA_ECC_FAMILY_SECP_R1));
 	psa_set_key_algorithm(&attr, PSA_ALG_ECDSA(PSA_ALG_SHA_256));
 	alg = PSA_ALG_ECDSA(PSA_ALG_SHA_256);
-#else  /* CONFIG_JWT_SIGN_RSA */
+#else  /* CONFIG_JWT_SIGN_RSA_PSA */
 	psa_set_key_type(&attr, PSA_KEY_TYPE_RSA_KEY_PAIR);
 	psa_set_key_algorithm(&attr, PSA_ALG_RSA_PKCS1V15_SIGN(PSA_ALG_SHA_256));
 	alg = PSA_ALG_RSA_PKCS1V15_SIGN(PSA_ALG_SHA_256);
-#endif /* CONFIG_JWT_SIGN_ECDSA || CONFIG_JWT_SIGN_RSA */
+#endif /* CONFIG_JWT_SIGN_ECDSA_PSA || CONFIG_JWT_SIGN_RSA_PSA */
```

Comment:
```suggestion
#endif /* CONFIG_JWT_SIGN_ECDSA_PSA */
```

Reply:
Well this was related to the fact that I wrote something like:
```
#if defined(CONFIG_JWT_SIGN_ECDSA_PSA)
...
#else  /* CONFIG_JWT_SIGN_RSA_PSA */
...
#endif /* CONFIG_JWT_SIGN_ECDSA_PSA || CONFIG_JWT_SIGN_RSA_PSA */
```
The idea was to state the you surely picked one of the two. But since this created confusion I will fix this

Reply:
Yeah I understood that, but it's rather done this way:
```
#if FIRST_COND
[...]
#else /* SOME_OTHER_COND */
[...]
#endif /* FIRST_COND */
```
Not saying it's the best, but just for consistency across the codebase.